### PR TITLE
Improve logging for locked file acquisition

### PR DESCRIPTION
**Pull request by zanieb**
Consistency etc. when debugging a deadlock

### CODE REVIEW

File: crates/uv-fs/src/lib.rs [hunk 1]
```diff
@@ -374,6 +384,8 @@ impl Drop for LockedFile {
                 self.0.path().display(),
                 err
             );
+        } else {
+            debug!("Released lock at `{}`", self.0.path().display());
```
Comment (charliermarsh) [inline crates/uv-fs/src/lib.rs:388]: Should this be `trace!` like the above?

File: crates/uv-fs/src/lib.rs [hunk 2]
```diff
@@ -316,27 +316,37 @@ pub struct LockedFile(fs_err::File);
...
-                        format!("Could not lock {}: {}", file.path().user_display(), err),
+                        format!(
+                            "Could not acquire lock for `{resource}` at `{}`: {}",
+                            file.path().user_display(),
+                            err
+                        ),
                     )
                 })?;
+
+                debug!("Acquired lock for `{resource}`");
```
Comment (charliermarsh) [inline crates/uv-fs/src/lib.rs:349]: Should this be `trace!` like the above?

File: crates/uv-fs/src/lib.rs (hunk 1 above)
Comment (zanieb) [reply crates/uv-fs/src/lib.rs:388]: Maybe since it comes after all the other output.

File: crates/uv-fs/src/lib.rs (hunk 2 above)
Comment (zanieb) [reply crates/uv-fs/src/lib.rs:349]: I don't think so — the acquired lock message above is debug level. The trace logs are for the acquire lock error message and the check message.

Comment (charliermarsh) [reply crates/uv-fs/src/lib.rs:349]: It still feels like a lot for `--verbose` IMO.

Comment (zanieb) [reply crates/uv-fs/src/lib.rs:349]: That's fine with me I think, but we'll want to change both.

**Pull request merged**